server/channels/store/storetest/settings.go

```diff
@@ -260,14 +243,13 @@ func MakeSqlSettings(driver string, withReplica bool) *model.SqlSettings {
 		panic("unsupported driver " + driver)
 	}
 
-	log("Created temporary " + driver + " database " + dbName)
```

Comment:
Can we keep this? Sometimes I need to see what database got created during a test, so that I can login to the DB to inspect some data or change something. This is especially helpful during a flaky test.

Reply:
Given we drop these databases at the end of the package tests, do you do something unique to preserve them? Wondering if we can log only when we expect a lifetime longer than the test invocation. Maybe we need a `MM_TEST_KEEP_DATABASE` or the like?

I'd love to omit superfluous logs in the happy path.

Reply:
>  do you do something unique to preserve them?

Yes, I use dlv to start my tests. And then step through the code. In that case, I can easily see what database to log in to from the logs.